src/test/FuzzerImpl.cpp

```diff
@@ -893,9 +910,20 @@ TransactionFuzzer::initialize()
                             toMuxedAccount(account);
                         ops.emplace_back(reduceNonNativeBalanceOp);
                     }
-                    // Here we could also reduce account "i"'s trust line limit
-                    // for asset "j" (testing the buying liabilities to
-                    // determine by how much).
+
+                    // Reduce "account"'s trustline limit for asset "j".
```

Comment:
I'm not sure about this for the same reason I'm unsure about the trust line flags in an earlier commit. In both places, you are implicitly assuming uniformity and the implementation doesn't permit anything else. Given that I think uniformity is unlikely what we desire (although in what ways it shouldn't be uniform will be for you to determine) I'm not sure how much value there is in these implementations.

Reply:
Yeah, I thought it might be more readable if I did that step first and then after getting that in replaced the single limits with per-asset arrays in `AccountParameters`.  If that's a sensible approach, I could either remove this step from the PR altogether and save it all for a future PR, update the commit to do all that in one step in this PR, or add a further commit to this one to take that further step.  (Even after that, it would be possible to take another step afterwards to make the issuance of `Asset`s more flexible -- currently the number and codes are the same for each issuer.)  Do you have a preference?